constance/test/pytest.py

```diff
@@ -0,0 +1,79 @@
+"""
+Pytest constance override config plugin.
+
+Inspired by https://github.com/pytest-dev/pytest-django/.
+"""
+import pytest
+from contextlib import ContextDecorator
+from constance import config as constance_config
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_configure(config):  # pragma: no cover
+    """
+    Register override_config marker.
+    """
+    config.addinivalue_line(
+        "markers",
+        (
+            "override_config(**kwargs): "
+            "mark test to override django-constance config"
+        )
+    )
+
+
+@pytest.hookimpl(hookwrapper=True)
+def pytest_runtest_call(item):  # pragma: no cover
+    """
+    Validate constance override marker params. Run test with overrided config.
+    """
+    marker = item.get_closest_marker("override_config")
```

Comment:
Need something like [here (in pytest-django)](https://github.com/pytest-dev/pytest-django/blob/ee7858af1a80e0091af4d260c0a70e5766c1196a/pytest_django/plugin.py#L421) to prevent #410 
```suggestion
    if not django_settings_is_configured():
        return
    marker = item.get_closest_marker("override_config")
```